src/js/renderers/editor-dom.js

```diff
@@ -91,6 +93,21 @@ function renderCard() {
   return { wrapper, cardElement };
 }
 
+function renderAtom(element, previousRenderNode) {
+  let atomElement = document.createElement('span');
+  addClassName(atomElement, 'ck-atom');
```

Comment:
do we need a class name here?

At the least let's make it `-mobiledoc-kit__atom`